cluster/saltbase/salt/kube-apiserver/initd

```diff
@@ -38,6 +38,9 @@ DAEMON_USER=kube-apiserver
 #
 do_start()
 {
+        # Raise the file descriptor limit - we expect to open a lot of sockets!
+        ulimit -n 65536
```

Comment:
why 65536 specifically (I've only played with ulimit lightly, so this might be a dumb question)?